multichain-testing/tools/purse.ts

```diff
@@ -1,10 +0,0 @@
-import type { Amount, Brand } from '@agoric/ertp';
-const { fromEntries } = Object;
-
-// @ts-expect-error Type 'Brand' does not satisfy the constraint 'string | number | symbol'
```

Comment:
I ran into this recently: https://github.com/Agoric/agoric-sdk/pull/10713/files#diff-a54584fae7da5725f7cb10d222f2eb764b7a9d51256e646ebb8dab4af9a0647fR121

thanks for addressing